homeassistant/components/contec_controllers/binary_sensor.py

```diff
@@ -0,0 +1,68 @@
+"""Contec Pusher."""
```

Comment:
There are multiple platforms in this first PR. Please limit the integration to a single platform.

See: https://developers.home-assistant.io/docs/creating_component_code_review/#5-make-your-pull-request-as-small-as-possible

Reply:
Thank you @frenck  for reviewing my PR.
I removed the cover and binary sensor platforms, an left with light only.